opm/grid/cpgrid/CpGridData.cpp

```diff
@@ -2263,5 +2263,56 @@ CpGridData::refinePatch(const std::array<int,3>& cells_per_dim, const std::array
 }
 
 
+std::array<double,3> CpGridData::getAverageArr(const std::vector<std::array<double,3>>& vec) const
+{
+    assert(!vec.empty());
+    std::array<double,3> result = {0., 0., 0.};
+    for (int coord = 0; coord < 3; ++coord) {
+        for (const auto& arr : vec) {
+            result[coord] += (arr.at(coord))/static_cast<double>(vec.size());
```

Comment:
I think we should make this as fast as possible and use `arr[coord]` as `at` will do some additional checks. I guess we can be sure here that the entries are there as we have put them there.

Another option would be to move the division outside of the loops. Not 100% sure whether that will be faster, though.

Reply:
Thanks! at() is replaced by [ ] now.